manifests/07-support-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
This would need to have the opt in settings.  Since we already have config files in the deployed yaml and the CI systems are already deploying the secret, you shouldn't need this.

Once we have the support config, we'll create a default support object with a boolean for enabled / disabled, which will then remove the need for the secret.

Reply:
I removed the secret manifest and cleaned up the commit message.  I went ahead and squashed my commits as well.